KarhooSDK/Service/Quote/QuoteService.swift

```diff
@@ -12,4 +12,5 @@ public protocol QuoteService {
     func quotes(quoteSearch: QuoteSearch) -> PollCall<Quotes>
     func coverage(coverageRequest: QuoteCoverageRequest) -> Call<QuoteCoverage>
     func verifyQuote(verifyQuotePayload: VerifyQuotePayload) -> Call<Quote>
+    func getVehiclesRules() -> Call<VehicleRules>
```

Comment:
I think `getVehicleImageRules` would be more descriptive. The rules are specific to the vehicle image

Reply:
👍  changed all names to include `Image` word.